### ISSUE TITLE
lack of accessibility in the W3 information video about accessibility

### ISSUE THREAD

**Issue by notSoEasyUser** [w3c/wai-easy-checks#118]
It is really annoying to have background music in the video
Easy Checks Overview found
https://www.w3.org/WAI/test-evaluate/preliminary/#video
because it makes it much harder to understand what is said. In my case  background music is more than an annoyance. I cannot understand the text at all.

**Comment by shawna-slh** [comment]
Your comment came at a good time, as we are creating new videos and it is good to have more firsthand accounts of the problems with background music.

Here are two different options to get the text:
1. Under the video, select the [+] before "Text Transcript with Description of Visuals". That will display a table. The first column of the table is the text.
2. In the video controls, click the fourth icon that looks like a page. It will bring up the transcript along with the video.
